src/lib/index.js

```
@@ -2,84 +2,84 @@
-        const r = Object.getOwnPropertyDescriptor(n, t);
+        const o = Object.getOwnPropertyDescriptor(r, t);
         Object.defineProperty(
           e,
           t,
-          r.get ? r : { enumerable: !0, get: () => n[t] },
+          o.get ? o : { enumerable: !0, get: () => r[t] },
         );
       }
   }
-  return (e.default = n), Object.freeze(e);
+  return (e.default = r), Object.freeze(e);
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)